--- a/mt_wifi/embedded/include/gas.h
+++ b/mt_wifi/embedded/include/gas.h
@@ -153,13 +153,13 @@
 			UCHAR DialogToken;
 			UCHAR AdvertisementProID;
 			UINT16 QueryReqLen;
-			UCHAR QueryReq[0];
+			UCHAR QueryReq[];
 		} GNU_PACKED GAS_REQ_DATA;
 		struct {
 			UINT16 StatusCode;
 			UCHAR AdvertisementProID;
 			UINT16 QueryRspLen;
-			UCHAR QueryRsp[0];
+			UCHAR QueryRsp[];
 		} GNU_PACKED PEER_GAS_RSP_DATA;
 		struct {
 			UCHAR DialogToken;
@@ -168,7 +168,7 @@
 			UINT16 StatusCode;
 			UCHAR AdvertisementProID;
 			UINT16 QueryRspLen;
-			UCHAR QueryRsp[0];
+			UCHAR QueryRsp[];
 		} GNU_PACKED GAS_CB_RSP_DATA;
 		struct {
 			UCHAR DialogToken;
@@ -182,7 +182,7 @@
 			UINT16 GASComebackDelay;
 			UCHAR AdvertisementProID;
 			UINT16 QueryRspLen;
-			UCHAR QueryRsp[0];
+			UCHAR QueryRsp[];
 		} GNU_PACKED GAS_RSP_DATA;
 		struct {
 			UCHAR DialogToken;
@@ -194,7 +194,7 @@
 			UCHAR DialogToken;
 			UCHAR AdvertisementProID;
 			UINT16 QueryReqLen;
-			UCHAR QueryReq[0];
+			UCHAR QueryReq[];
 		} GNU_PACKED PEER_GAS_REQ_DATA;
 		struct {
 			UCHAR DialogToken;
